plyer/platforms/linux/orientation.py

```diff
@@ -0,0 +1,27 @@
+from subprocess import call, check_output
+from plyer.facades import Orientation
+from plyer.utils import whereis_exe
+
+
+class XrandrOrientation(Orientation):
+
+    def _set_landscape(self, **kwargs):
+        self.rotate = 'normal'
+        self.screen = check_output("xrandr -q | grep ' connected' \
+                              |  head -n 1 | cut -d ' ' -f1", shell=True)
```

Comment:
i have dual monitor and this works for single monitor only.For windows i remember i was working it out with some win32api ,i think i have it in my local repo but could not push it bcz i was unable to test it on windows (i have linux :( ).For mac i was using some external library as of i recal some thing like fb-rotate.